src/tiledimage.js

```diff
@@ -1148,38 +1150,41 @@ function compareTiles( previousBest, tile ) {
     return previousBest;
 }
 
-function drawTiles( tiledImage, lastDrawn ){
+function drawTiles( tiledImage, lastDrawn ) {
     var i,
         tile,
         tileKey,
         viewer,
         viewport,
         position,
-        tileSource;
+        tileSource,
+        contextSaved = false;
```

Comment:
This should be changed back to `usedClip` or the `usedClip` below should be changed to `contextSaved`.